docs/developer-guide.md

```diff
@@ -73,7 +73,47 @@ On Windows the file must be renamed to `djinni.bat` to make it executable.
 
 ## Project Structure
 
-!!! bug "TODO"
+```text
+.
+├── CODE_OF_CONDUCT.md (1)
+├── LICENSE (2)
+├── README.md (3)
+├── build.sbt (4)
+├── docs (5)
+│   └── ...
+├── mkdocs.yml (6)
+├── project (7)
+│   └── ...
+└── src (8)
+    ├── it (9)
+    │   ├── resources (10)
+    │   │   ├── expected (11)
+    │   │   │   └── ...
+    │   │   └── result (12)
+    │   │       └── ...
+    │   └── scala 
+    │       └── djinni (13)
+    │           └── ...
+    └── main
+        └── scala
+            └── djinni (14)
+                └── ...
+```
+
+1. Project Code of Conduct.
+2. Project License (Apache License).
```

Comment:
Do you think this is helpful? I think it may help to get an overview of the project structure...

Reply:
Yes, this helps for sure!